src/ioctls/vm.rs

```diff
@@ -247,7 +247,7 @@ impl VmFd {
     }
 
     /// Get the `kvm_run` size.
-    pub fn get_run_size(&self) -> usize {
+    pub fn run_size(&self) -> usize {
```

Comment:
This change should be a separate commit. Maybe even a separate PR.

Reply:
I'll add it to a different commit (I was lazy).

The merge commit will vanish once we merge the PR to master. That's how we also do it in Firecracker to avoid `push -f` when the only thing you want to do is update the branch to master.

Trying to find an official documentation that explains this but totally failing.

Reply:
> avoid push -f when the only thing you want to do is update the branch to master.

I figured that much. I personally don't mind push forcing to a development branch in order to keep the reviewed commit history clean.